src/components/robot/RobotEdit.tsx

```
@@ -133,15 +149,19 @@ export const RobotEditModal = ({ isOpen, handleStart, handleClose, initialSettin
                     action.args?.length >= 2 && 
                     typeof action.args[1] === 'string'
                 ) {
-                    let currentSelector: string = action.args[0];
-                    let character: string = action.args[1];
+                    const currentSelector: string = action.args[0];
+                    const character: string = action.args[1];
+                    const inputType: string = action.args[2] || '';
                     
                     if (!credentials.hasOwnProperty(currentSelector)) {
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)